src/sass/_Fabric.Typography.Mixins.scss

```diff
@@ -5,7 +5,7 @@
 // --------------------------------------------------
 // Fabric Core Typography mixins
 
-@mixin ms-baseFont() {
+@mixin ms-baseFont {
   font-family: $ms-font-family-base;
```

Comment:
Totally agreed. The only reason I'm questioning it now is that some teams consuming Fabric right now use separate definitions/font family names for Segoe than what Fabric currently provides, so this would produce work for them to override. Happy to discuss though--this definitely moves us in the right direction.